src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx

```diff
@@ -5,27 +5,38 @@ import ClaimsUnavailable from '../components/ClaimsUnavailable';
 import ClaimSyncWarning from '../components/ClaimSyncWarning';
 import RequiredLoginView from '../../common/components/RequiredLoginView';
 
-class DisabilityBenefitsApp extends React.Component {
+// this needs to be a React component for RequiredLoginView to pass down props
+function AppContent({ available, synced, children, isDataAvailable }) {
+  // prop is only passed on failure
+  const canUseApp = isDataAvailable === true || typeof isDataAvailable === 'undefined';
```

Comment:
@crwallace It might be useful to pass this prop all the time, to avoid checks like this (unless I'm missing something).